### PR TITLE
Initial pysat-LIBS changes

### PR DESCRIPTION
**Pull request by RyanA1084**
This updates the autocnet spectral files to include changes and additions made by Ryan for pysat-LIBS. Most changes have to do with I/O for laboratory and ChemCam data files.

### CODE REVIEW

File: autocnet/fileio/io_csv.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    df_spectra.columns=pd.MultiIndex.from_tuples(cols_wvl)
+    for i,x in enumerate(df_data.columns):
+        df_spectra[x]=df_data[x]
+    df=df_spectra
+
+    if setindex:
+        df=df.set_index([setindex])
+
+
+    return df
```
Comment (jlaura) [inline autocnet/fileio/io_csv.py]: Why not use the Pandas `read_csv`?

File: scratch.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+#
+#edrtest=r"C:\Users\rbanderson\Documents\Projects\LIBS PDART\pysat\pysat\examples\ChemCam\CL5_399178818EDR_F0030078CCAM01019M1_spect.TXT"
+#edr=EDR(edrtest)
+#
+
+
+#maskfile=r"C:\Users\rbanderson\Documents\Projects\MSL\ChemCam\DataProcessing\Working\Input\mask_minors_noise.csv"
+#masksav=sav.mask(sav,maskfile)
+
+print("foo")
```
Comment (jlaura) [inline scratch.py]: What is this file?  Is it intentionally added?

Comment (RyanA1084) [inline scratch.py]: Oops, we can leave this one out. It's just a scratch file I was using to test the different functions.

File: autocnet/fileio/io_csv.py (hunk 1 above)
Comment (RyanA1084) [inline autocnet/fileio/io_csv.py]: This uses read_csv but then does some manipulation of the column names to identify columns where the column name is numeric (the wavelength of the spectrometer channel) and flag them as spectral data, allowing the spectral data to be isolated in the final data frame by just using df['wvl']. I'll add some comments and change the file name to io_csv_libs.py to clarify the purpose.

**Comment by jlaura** [comment]
I can't update the pull request with the changes.  If you need some help with how to update, let me know.

**Comment by jlaura** [comment]
Also, if you `git fetch upstream master`, `git checkout master`, `git merge upstream/master`, `git push origin master`, this PR will be updated with changes made to the `.travis.yml` file to get CI working.

File: autocnet/fileio/io_csv.py (hunk 1 above)
Comment (RyanA1084) [inline autocnet/fileio/io_csv.py]: Actually, upon thinking about this more, you're right. read_csv can do what I need as long as the user formats the csv file correctly.

**Comment by jlaura** [comment]
:+1:

**Comment by jlaura** [comment]
As a total aside - any ideas for a name for this thing?  We have spectral plus image matching in here.  Lots more to add to expand functionality into a broader planetary library.

**Comment by RyanA1084** [comment]
Here are some ideas off the top of my head:
PyPlanet
PlanetPy
PlanetTools
PyISIS (PySIS?)

**Comment by thareUSGS** [comment]
PlanetaryPy taken. https://github.com/planetarypy
Planet Python: http://planetpython.org/
PyPlanet taken: https://github.com/david-deboer/pyplanet,
https://github.com/MateusZitelli/PyPlanets

Perhaps, we can think about "forking" this under PlanetaryPY site once it
is more stable. So maybe a good name that would work under that title?

Jay - Maybe for reading ISIS3 cubes (if you don't need all the spatial
cruft) you can skip GDAL and use his reader :-)
https://github.com/planetarypy/planetaryimage

-Trent

**Pull request merged**